client/components/UI/Form/TimeInput/index.tsx

```diff
@@ -77,7 +85,15 @@ export class TimeInput extends React.Component {
             value.format(appConfig.planning.timeformat) :
             (value || '');
 
-        this.setState({viewValue});
+        this.setState({viewValue}, this.updateInputWidth);
+
+        if (this.dom.inputField) {
+            this.inputPadding = parseFloat(window.getComputedStyle(this.dom.inputField).paddingInline) * 2;
```

Comment:
Why are you multiplying by 2? Can you get the properties that you need directly? maybe paddingInlineStart + paddingInlineEnd ?